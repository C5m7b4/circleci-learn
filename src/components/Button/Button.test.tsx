import React from 'react';
import { render, fireEvent } from '@testing-library/react';

import Button from './Button';

describe('Button', () => {
  test('should render correctly', () => {
    const testFn = jest.fn();
    const { container } = render(
      <Button onClick={testFn} label={'Click Me'} />
    );
    expect(container).toMatchSnapshot();
    const btn = fireEvent.click(container);
  });
});
