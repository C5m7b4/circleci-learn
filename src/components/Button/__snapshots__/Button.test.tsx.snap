// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render correctly 1`] = `
<div>
  <button
    data-testid="btn"
  >
    Click Me
  </button>
</div>
`;
